app/controllers/ncr/work_orders_controller.rb

```diff
@@ -4,7 +4,7 @@ class WorkOrdersController < ClientDataController
     MAX_UPLOADS_ON_NEW = 10
 
     def new
-      work_order.approving_official_email = suggested_approver_email
+      @client_data_instance.approving_official = suggested_approver_email
```

Comment:
ah yes I changed this when debugging something and forgot to change back. I am happy to change back or change everywhere - which do you think is clearer?